src/main/java/com/cloudbees/jenkins/GitHubCommitNotifier.java

```diff
@@ -34,9 +42,16 @@
  */
 public class GitHubCommitNotifier extends Notifier {
 
+    private ExpandableMessage statusMessage = new ExpandableMessage("");
```

Comment:
So if all plugins do such things, i will have fully empty configuration that i should click on advanced/help buttons to get what is really will be during execution.